Source/gg2/Objects/InGameElements/Character.events/Begin Step.xml

```diff
@@ -311,7 +323,7 @@ if (intel)
     if (cloak)
     {
         cloak=false;
-        cloakAplha=1;
+        cloakAplha=1; // FIXME: Is this typo causing a bug? Will fixing the typo cause a bug?
```

Comment:
It seems to do pretty much nothing, because you cannot pick up the intel while cloaked. The only reason I see for it is to avoid a spy from being invisible client-side due to desync while carrying the intel. I fixed the typo now because eh.